app/controllers/healthcheck_controller.rb

```diff
@@ -13,7 +13,7 @@ def check
         migration_version: migration_version,
       },
       delayed_job: {
```

Comment:
This should either become sidekiq, or we should add a new section. Probably the latter, while we run them in parallel